ios/Handlers/RNManualHandler.m

```diff
@@ -35,6 +35,16 @@ - (void)touchesMoved:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event
 {
   [super touchesMoved:touches withEvent:event];
   [_gestureHandler.pointerTracker touchesMoved:touches withEvent:event];
+
+  if ([self shouldFail]) {
+    if (_gestureHandler.state == RNGestureHandlerStateActive) {
+      self.state = UIGestureRecognizerStateFailed;
+    } else if (_gestureHandler.state == RNGestureHandlerStateBegan) {
+      self.state = UIGestureRecognizerStateCancelled;
+    }
```

Comment:
It may be better to stay consistent and rely on the `UIGestureRecognizer`'s state here, like so: https://github.com/software-mansion/react-native-gesture-handler/blob/234ec684e8cc3e6e65a2e42f5ed3ae72c4d6e681/ios/Handlers/RNPanHandler.m#L122-L124

Reply:
Sure, I've missed that I left `RNGestureHandler` states in ifs